developer/src/tike/project/Keyman.Developer.UI.Project.UfrmNewProjectParameters.pas

```diff
@@ -353,7 +361,8 @@ procedure TfrmNewProjectParameters.EnableControls;
   e := (Trim(editKeyboardName.Text) <> '') and
     (Trim(editPath.Text) <> '') and
     TKeyboardUtils.IsValidKeyboardID(Trim(editKeyboardID.Text), True) and
-    (GetTargets <> []);
+    (GetTargets <> []) and
+    (Trim(memoDescription.Text) <> '');
```

Comment:
I was thinking
```suggestion
    (GetDescription <> '');
``` 
would be more DRY. But then it would involve changing the other ones, and then
I saw GetKeyboardID returns a Trim(**LowerCase**(editKeyboardID.Text)), so maybe nvm